lib/controller_patches.rb

```diff
@@ -52,7 +52,7 @@ def blog_cache(cache_key, expires=4.hours)
       # if the fragment is unreadable or has reached expiry
       # attempt to pull in the blog feed
       if updated.nil? || !fragment_exist?(@fragment_key) ||
-         Time.now > (Time.parse(updated) + expires)
+         Time.zone.now > (Time.parse(updated) + expires)
```

Comment:
It might be worth changing the `Time.parse` to `Time.zone.parse` as well

Reply:
Ah yep, done now